### ISSUE TITLE
Initial learning rate after restoring the model from ckpt

### ISSUE THREAD

**Issue by AlexDenis** [rishizek/tensorflow-deeplab-v3#18]
Hello once again! 
I was wondering If there is an easy way to continue training the model using the initial learning rate (and decreasing it to the end learning rate using poly/piecewise policy) after restoring the model from my previous checkpoint? As far as I understand, right now training continues with the same learning rate as it was using in the moment of checkpoint creation. Thanks!
Also, have you tried to implement adaptive learning rate? I'm currently training the model on COCO Panoptic dataset and it feels that it takes some dark magic to handcraft proper learning rates (initial, ending and total number of iterations to get from one to another).
